virtual_rainforest/models/animals/constants.py

```diff
@@ -10,13 +10,18 @@
 """  # noqa: D205, D415
 
 
-ENDOTHERM_METABOLIC_RATE_TERMS: dict[str, tuple[float, float]] = {
-    "mammal": (0.75, 0.047),
-    # Mammalian herbivore population density, observed allometry (Damuth 1987). [kg]
+ENDOTHERMIC_METABOLIC_RATE_TERMS: dict[str, tuple[float, float]] = {
+    "mammal": (0.75, 0.018),
+    # Mammalian metabolic rate scaling from Metabolic Ecology p210. [g]
     "bird": (0.75, 0.05),
     # Toy values.
 }
 
+ECTOTHERMIC_METABOLIC_RATE_TERMS: dict[str, tuple[float, float]] = {
+    "insect": (0.75, 0.08)
+    # Insect metabolic rate scaling from Metabolic Ecology p210. [g]
```

Comment:
Sloppy comment on my part. Some of those unit references are actually reminders to myself about what mass unit the source is assuming. It's all handled in scaling_functions right now but it will save me a lot of annoying time when I am ironing out the parameters later on. I have just added a commit that clarifies what those unit references actually are.